components/LessonCard.tsx

```diff
@@ -73,9 +73,11 @@ const LessonCard: React.FC<Props> = props => {
           objectFit="contain"
         />
 
-        <div className={`${styles['lesson-card__description']} pl-4`}>
+        <div
+          className={`${styles['lesson-card__description']} pl-4 d-flex flex-column d-md-block`}
+        >
           {props.currentState === 'completed' && (
-            <span className="badge badge-pill badge-success float-right mt-2 mr-2 p-2 d-flex align-items-center">
+            <span className="badge badge-pill badge-success float-right align-self-end mt-2 mr-2 p-2 d-flex align-items-center">
               <CheckCircle style={{ height: '15px' }} />
               <span className="mx-1">COMPLETED</span>
```

Comment:
Bootstrap utilities like 
`<span className="mx-1 d-none d-md-inline">COMPLETED</span>`
Can be used to hide this and only leave the check on smaller screen
![image](https://user-images.githubusercontent.com/20666236/124374035-78465a80-dcb5-11eb-999c-ffdb69ba3844.png)